src/proposals/20231115/SparkEthereum_20231115.sol

```diff
@@ -0,0 +1,159 @@
+    address public constant DAI              = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    address public constant WETH             = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address public constant WBTC             = 0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599;
+    address public constant GNOSIS_FORWARDER = 0x44f993EAe9a420Df9ffa5263c55f6C8eF46c0340;
+    address public constant GNOSIS_PAYLOAD   = 0xF62849F9A0B5Bf2913b396098F7c7019b51A820a; // Temporary test address, to be replaced after deployment
+
+    function capsUpdates()
+        public pure override returns (IEngine.CapsUpdate[] memory)
+    {
+        IEngine.CapsUpdate[] memory capsUpdate = new IEngine.CapsUpdate[](3);
+        capsUpdate[0] = IEngine.CapsUpdate({
+            asset:     RETH,
+            supplyCap: 80_000,
+            borrowCap: EngineFlags.KEEP_CURRENT
+        });
+
+        capsUpdate[1] = IEngine.CapsUpdate({
+            asset:     WSTETH,
+            supplyCap: 800_000,
+            borrowCap: EngineFlags.KEEP_CURRENT
+        });
+
+        capsUpdate[2] = IEngine.CapsUpdate({
+            asset:     WBTC,
+            supplyCap: 3_000,
+            borrowCap: 2_000
+        });
+
+        return capsUpdate;
+    }
+
+    function collateralsUpdates()
+        public pure override returns (IEngine.CollateralUpdate[] memory)
+    {
+        IEngine.CollateralUpdate[] memory collateralUpdates = new IEngine.CollateralUpdate[](2);
+
+        collateralUpdates[0] = IEngine.CollateralUpdate({
+            asset:          DAI,
+            ltv:            0,
+            liqThreshold:   EngineFlags.KEEP_CURRENT,
+            liqBonus:       EngineFlags.KEEP_CURRENT,
+            debtCeiling:    EngineFlags.KEEP_CURRENT,
+            liqProtocolFee: EngineFlags.KEEP_CURRENT,
+            eModeCategory:  EngineFlags.KEEP_CURRENT
+        });
+
+        collateralUpdates[1] = IEngine.CollateralUpdate({
+            asset:          WBTC,
+            ltv:            70_00,
+            liqThreshold:   75_00,
+            liqBonus:       7_00,
+            debtCeiling:    0,
+            liqProtocolFee: 10_00,
+            eModeCategory:  0
+        });
+
+        return collateralUpdates;
+    }
+
+    function borrowsUpdates()
+        public pure override returns (IEngine.BorrowUpdate[] memory)
+    {
+        IEngine.BorrowUpdate[] memory borrowsUpdate = new IEngine.BorrowUpdate[](1);
+
+        borrowsUpdate[0] = IEngine.BorrowUpdate({
+            asset:                 WBTC,
+            reserveFactor:         20_00,
+            enabledToBorrow:       EngineFlags.ENABLED,
+            flashloanable:         EngineFlags.ENABLED,
+            stableRateModeEnabled: EngineFlags.DISABLED,
+            borrowableInIsolation: EngineFlags.DISABLED,
+            withSiloedBorrowing:   EngineFlags.DISABLED
+        });
+
+        return borrowsUpdate;
+    }
+
+    function rateStrategiesUpdates()
```

Comment:
Can we use this syntax instead? Was used in almost all past spells for rates updates

```solidity
    function rateStrategiesUpdates()
        public view override returns (IEngine.RateStrategyUpdate[] memory)
    {
        IEngine.RateStrategyUpdate[] memory ratesUpdate = new IEngine.RateStrategyUpdate[](1);

        Rates.RateStrategyParams memory weth = LISTING_ENGINE
            .RATE_STRATEGIES_FACTORY()
            .getStrategyDataOfAsset(WETH);

        weth.optimalUsageRatio  = NEW_WETH_OPTIMAL_USAGE_RATIO;
        weth.variableRateSlope1 = NEW_WETH_VARIABLE_RATE_SLOPE_1;
        weth.variableRateSlope2 = NEW_WETH_VARIABLE_RATE_SLOPE_2;

        ratesUpdate[0] = IEngine.RateStrategyUpdate({
            asset:  WETH,
            params: weth
        });

        return ratesUpdate;
    }
```

Just out of an abundance of caution not to overwirte to zero.